frontend/src/features/admin/users/ManageUsersPage.tsx

```diff
@@ -62,6 +62,13 @@ export const ManageUsersPage = () => {
     state => (state.network as any)[actionTypes.GET_USERS] as IGenericNetworkAction,
   );
 
+  const onRequestData = useCallback(
```

Comment:
callbacks are required for functions that control react-table behaviour (ie. pageIndex, pageSize, etc). Otherwise these functions run on every render of the table.